### PR TITLE
[improvement](executor)Add name for task scheduler

### PR DESCRIPTION
**Pull request by wangbo**
## Proposed changes

Add name for task scheduler, then we tell different thread pool in TOP command.

### CODE REVIEW

**Comment by wangbo** [comment]
run buildall

File: be/src/runtime/exec_env_init.cpp [hunk 1]
```diff
@@ -229,12 +229,14 @@ Status ExecEnv::init_pipeline_task_scheduler() {
     // TODO pipeline task group combie two blocked schedulers.
     auto t_queue = std::make_shared<pipeline::MultiCoreTaskQueue>(executors_size);
     auto b_scheduler = std::make_shared<pipeline::BlockedTaskScheduler>(t_queue);
-    _pipeline_task_scheduler = new pipeline::TaskScheduler(this, b_scheduler, t_queue);
+    _pipeline_task_scheduler = new pipeline::TaskScheduler(this, b_scheduler, t_queue,
+                                                           "TaskSchedulerThreadPoolWithoutGroup");
```
Comment (yiguolei) [inline be/src/runtime/exec_env_init.cpp:233]: The name is too long, maybe not display with top -H

**Comment by wangbo** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**